android/tangram/src/com/mapzen/tangram/MapData.java

```diff
@@ -0,0 +1,102 @@
+package com.mapzen.tangram;
+
+import java.util.List;
+
+public class MapData {
+
+    /**
+     * Construct a new MapData object for adding drawable data to the map
+     * @param name Name of the data source in the scene file for styling this object's data
+     */
+    public MapData(String name) {
+        this.name = name;
+        this.id = addDataSource(name);
+    }
+
+    /**
+     * Get the name of this data source
+     * @return The name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Remove all data from this source
+     * @return This object, for chaining
+     */
+    public MapData clear() {
+        clearSourceData(id);
+        return this;
```

Comment:
```java
mapData.clear()
       .addPoint(p0)
       .addPoint(p1);
```

Some people prefer chain-style calls. It's not like we have anything better to return :P